src/comlink.ts

```diff
@@ -28,6 +28,24 @@ export const releaseProxy = Symbol("Comlink.releaseProxy");
 
 const throwMarker = Symbol("Comlink.thrown");
 
+/**
+ * Imports required to support running in NodeJS.
+ *
+ * See use of the replace rollup plugin in rollup.config.js.
+ *
+ * Unfortunately, TLA (top-level await) is not widely supported in browsers yet, and even leaving this conditional
+ * statement can yield errors (see Firefox bug https://bugzilla.mozilla.org/show_bug.cgi?id=1693570).
+ * So the current solution to import the `worker_threads` module only works for the umd bundle.  Once TLA is widely
+ * supported the dynamic import solution can be enabled.
+ */
+import nodeEndpoint, { NodeEndpoint } from "./node-adapter";
+let NodeMessageChannel: unknown = undefined;
```

Comment:
sorry about the `unknown` here - it sucks, but the two `MessageChannel` are incompatible :(

Reply:
Given we now have TLA support across all AAA browsers, I suspect we could simplify this too?